simuleval/test/test_s2s.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import tempfile
+from pathlib import Path
+
+import simuleval.cli as cli
+from simuleval.agents import SpeechToSpeechAgent
+from simuleval.agents.actions import ReadAction, WriteAction
+from simuleval.data.segments import SpeechSegment
+
+ROOT_PATH = Path(__file__).parents[2]
+
+
+def test_s2s(root_path=ROOT_PATH):
+    args_path = Path.joinpath(root_path, "examples", "speech_to_speech")
+    os.chdir(args_path)
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        cli.sys.argv[1:] = [
+            "--agent",
+            os.path.join(
+                root_path, "examples", "speech_to_speech", "english_alternate_agent.py"
+            ),
+            "--user-dir",
+            os.path.join(root_path, "examples"),
+            "--agent-class",
+            "agents.EnglishSpeechCounter",
+            "--source-segment-size",
+            "1000",
+            "--source",
+            os.path.join(root_path, "examples", "speech_to_speech", "source.txt"),
+            "--target",
+            os.path.join(root_path, "examples", "speech_to_speech", "reference/en.txt"),
+            "--output",
+            tmpdirname,
+        ]
+        cli.main()
+
+
+def test_statelss_agent(root_path=ROOT_PATH):
```

Comment:
nit: `test_stateless_agent` (I guess it's also misspelled in test_s2t, feel free to leave or fix it)

Reply:
Fixed!